layouts/_default/header.html

```diff
@@ -1,17 +1,25 @@
 <section id="header">
-    <div class="section-container">
-        <h1 class="title tagline">
-            {{ .Params.clinic }}<span class="hidden-phone">.</span>
-            <div class="linebreak visible-tablet"></div>
-            {{ .Params.services }}<span class="hidden-phone">.</span>
-            <div class="linebreak visible-tablet"></div>
-            {{ .Params.inventory }}<span class="hidden-phone">.</span>
-            <div class="linebreak visible-tablet"></div>
-            <span class="title-emphasis"> {{ .Params.online }}<span class="hidden-phone">.</span></span>
-        </h1>
-        <div class="section-content">
-            {{ .Content }}
-            {{ partial "startNowButton.html" "get-started-head" }}
-        </div>
+  <div class="section-container">
```

Comment:
It's kind of hard to read the pr. Lots of lines are changed, but only the indent changes in most cases. It's hard to rapidly have an overview of what has really changed